default.nix

```diff
@@ -31,7 +31,7 @@ in
 with nixpkgs;
 
 let
-    llvmForGhc = llvm_7;
+    llvmForGhc = nixpkgs.llvm_9;
```

Comment:
I think you can drop the `nixpkgs` prefix, because there's `with nixpkgs;` in line 31.

Reply:
Oops, indeed. Fixed, thanks!